osu.Game/Screens/Play/Player.cs

```diff
@@ -1238,7 +1238,13 @@ private void fadeOut(bool instant = false)
 
             if (this.IsCurrentScreen())
             {
-                ApplyToBackground(b => b.IgnoreUserSettings.Value = true);
+                ApplyToBackground(b =>
+                {
+                    b.IgnoreUserSettings.Value = true;
+
+                    b.IsBreakTime.UnbindFrom(breakTracker.IsBreakTime);
+                    b.IsBreakTime.Value = true;
```

Comment:
As far as I can tell, this is backwards, and judging from the OP:

> Can confirm that this was the issue by changing the `= false` that I added to an `= true`

it looks like a leftover change from that confirmation.